src/test/java/com/bio4j/angulillos/TwitterGraphTestSuite.java

```diff
@@ -14,8 +14,9 @@
 
   public TwitterSchema<RV,RE>.User.Vertex addUser(String name, Integer age) {
 
-    return g.addVertex(g.user)
-            .set(g.user.name, name)
+    TwitterSchema<RV,RE>.User.Vertex u = g.user.addVertex();
```

Comment:
The thing is that `VertexType<VT, ...>.Vertex` is the right type, once `VT` is fixed. Look at the "expected" type. I can write

```java
    TwitterSchema<RV,RE>.User.Vertex u1 = null; // just to have something of this type
    VertexType<TwitterSchema<RV,RE>.User, TwitterSchema<RV,RE>, RV,RE>.Vertex u2 = u1;
```

and another way round. And there won't be any warnings about unchecked conversions.